audmath/core/api.py

```diff
@@ -237,10 +251,18 @@ def to_nanos(value, unit):
 
     if isinstance(duration, str):
 
+        # none/-inf/inf duration
+        if duration == '' or duration == 'none':
```

Comment:
Maybe: `duration.lower() == 'none'`?

Reply:
Good question, I wasn't sure what `None` should be in string representation. If you provide `''` you don't have a value which felt similar to `None` to me. But I'm also fine with removing `''`

Reply:
I was actually referring to the second case since:

```
>>> str(None)
'None'
```

Reply:
Good point, I added `lower()` and also a test for `'None'`

Reply:
What about `NaN` / `nan`?

Reply:
I added support for `NaN` and `NaT` and any other lower/mixed case version of those words.

Reply:
Now that we support `None` we should probably express this in the type hint and change to `duration: typing.Optional[Union[float, int, str, np.timedelta64]],`

Reply:
And maybe also mention in the docstring that inf and nan is supported.

Reply:
I added `typing.Optional[]` and extended the docstring with:

![image](https://user-images.githubusercontent.com/173624/217206008-a2e8dc66-32f8-49a1-9009-0e5dd68bfad6.png)

Note, the link to `pandas.NaT` is not working, even though the link to `pandas.Timedelta` works, but I did not find a way to make `pandas.NaT` work, it seems to be some weird alias.

Reply:
Great, I think the only missing case is now `''`, which is also interpreted as `numpy.nan`.

Reply:
I added `''` as well to the docstring now.